apiary.apib

```diff
@@ -3352,7 +3348,310 @@ Updates and returns the updated setting object of the given id if the user owns
             "realtime_notifications":false
           }
         }
+## Group Space
+
+### Spaces [/spaces]
+
+#### Index [GET]
+
++ Request (application/json)
+
++ Response 200 (application/json)
+    + Attributes (object)
+        + spaces (array[Space]) - list of spaces
+
+    + Body
+
+            {
+              "spaces":[
+                {
+                  "id":"123456789",
```

Comment:
Because of some ambiguity in the description of the `Space` datatype, `id` should actually be of type `uuid`.